.golangci.yml

```diff
@@ -5,6 +5,8 @@ run:
 
 linters:
   default: all
+  enable:
+    - wsl_v5
```

Comment:
Do we need to enable this linter when we already enable all linters in the line above?